File: Pratica 2 - Estrutura de Decisao/cpp_logic/estruturadecisao.cpp
#include <iostream>
using namespace std;

int main(){
    int quant;
    cout << "Digite a quantidade em estoque: ";
    cin >> quant;

    if (quant < 1){
        cout << "Status: Em falta." << endl;
    } else if (quant < 5){
        cout << "Status: Alerta de estoque baixo." << endl;
    } else {
        cout << "Status: Estoque OK." << endl;
    }
    return 0;
}

File: Pratica 2 - Estrutura de Decisao/java_logic/Estruturadecisao.java
import java.util.Scanner;

public class Estruturadecisao {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite a quantidade em estoque: ");
        int quant = sc.nextInt();

        if (quant < 1) {
            System.out.println("Status: Em Falta.");
        } else if (quant < 5) {
            System.out.println("Status: Alerta de estoque baixo.");
        } else {
            System.out.println("Status: Estoque OK.");
        }
        sc.close();
    }
}
